config/locales/en.yml

```diff
@@ -45,6 +45,14 @@ en:
     oauth_text: "This account is authenticated through %{provider}."
     own_projects: "Projects you started"
     participating_projects: "Projects you are participating in"
+    projects: "Your projects"
+    no_projects: "There are no projects you participate in or organize."
```

Comment:
I'd be careful with the wording. AFAIK, the project never used the term "organize" with regard to the ownership of a project.